lib/ios/AirGoogleMaps/AIRGoogleMapPolyline.m

```diff
@@ -49,6 +52,39 @@ -(void)setStrokeWidth:(double)strokeWidth
   _strokeWidth = strokeWidth;
   _polyline.strokeWidth = strokeWidth;
 }
+-(void)setLineDashPattern:(NSArray <NSNumber *> *)lineDashPattern {
+  _lineDashPattern = lineDashPattern;
+}
+-(void)updateLineDashPattern:(double)scale {
+  
+  // scale = 1.0 / mapView.projection.pointsForMeters(1, atCoordinate: mapView.camera.target)
```

Comment:
Dead code

Reply:
What do you mean? this is what allows me to scale according to map movement/zoom and allows me to have the scale for when the line continues withouth moving the map

Reply:
Well, it's commented out now :) Commented code is dead code.

Reply:
ahh the line 60, forgot to remove that ahah